Neos.Flow/Classes/Http/Component/ComponentInterface.php

```diff
@@ -1,42 +0,0 @@
-<?php
```

Comment:
As discussed: I think we should keep the interface (deprecated) for b/c

Reply:
See https://discuss.neos.io/t/what-http-components-have-you-created-does-a-replacement-middleware-exists/5167/29
Martin is correct - we have both component chain and middleware chain in Flow 6.3, so both would run in that case. A workaround is probably... ugly to say the least?

Reply:
Thinking more about it: We'd need to somehow make 6.3 work with this case, not 7.0 - that means a bugfix release, which means any package can at best support `^6.3.4 || ^7.0`